x-pack/plugins/cases/public/components/case_form_fields/index.test.tsx

```diff
@@ -268,6 +278,7 @@ describe('CaseFormFields', () => {
             test_key_1: 'Test custom filed value',
             test_key_2: true,
             test_key_4: false,
+            test_key_5: 123,
```

Comment:
In the above test the value is a string but in this the value is a number. Shouldn't be both numbers?

Reply:
They are different, because in the first case we paste the number `await user.paste('4321');` and then it's string. But in the second test the number case from `defaultValue` which already was transformed to the number. 

Now I understand that article you gave me, where the guy is saying that number field are the worst)))